addons/skin.btplay/changelog.txt

```diff
@@ -0,0 +1,44 @@
+[B]0.8.5[/B]
```

Comment:
i don't think all of these files should have been added?

Reply:
Agreed, However I am up to date with develop. I'll look into it.

Reply:
Ok, It looks like my branch squashed your submodules even though they are listed in git.